### ISSUE TITLE
no pytorch_jni in java.library.path windows

### ISSUE THREAD

**Issue by beeqwe** [pytorch/pytorch#44363]
Hello, i'm trying to load my model using pytorch_java_only but it's return this error:

```
Exception in thread "main" java.lang.UnsatisfiedLinkError: no pytorch_jni in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at com.facebook.soloader.nativeloader.SystemDelegate.loadLibrary(SystemDelegate.java:22)
	at com.facebook.soloader.nativeloader.NativeLoader.loadLibrary(NativeLoader.java:43)
	at org.pytorch.NativePeer.<clinit>(NativePeer.java:11)
	at org.pytorch.Module.load(Module.java:23)
	at com.github.beeqwe.Bootstrap.main(Bootstrap.java:18)
```

How to reproduce the bug:

OS: **Windows**

```
public static void main(String[] args) throws IOException {
    System.setProperty("java.library.path", LIBTORCH_PATH + "lib");

    Module mod = Module.load(MODEL_PATH + "model.pt1"); //LINE 18
    Tensor data =
            Tensor.fromBlob(
                    new int[]{1, 2, 3, 4, 5, 6}, // data
                    new long[]{2, 3} // shape
            );
    IValue result = mod.forward(IValue.from(data), IValue.from(3.0));
    Tensor output = result.toTensor();

    System.out.println("shape: " + Arrays.toString(output.shape()));
    System.out.println("data: " + Arrays.toString(output.getDataAsFloatArray()));
}
```
pom.xml:

```
<dependency>
     <groupId>org.pytorch</groupId>
      <artifactId>pytorch_java_only</artifactId>
      <version>1.6.0</version>
</dependency>
<dependency>
    <groupId>com.facebook.fbjni</groupId>
    <artifactId>fbjni-java-only</artifactId>
    <version>0.0.3</version>
</dependency>
<dependency>
    <groupId>com.facebook.soloader</groupId>
    <artifactId>nativeloader</artifactId>
    <version>0.8.0</version>
</dependency>

```
**LIBTORCH_PATH** this is directory of files which downloaded from [there](https://download.pytorch.org/libtorch/cpu/libtorch-win-shared-with-deps-debug-1.6.0%2Bcpu.zip)

And i noticed one regularity, linux archive contains the libpytorch_jni.so file and macos archive contains the libpytorch_jni.dylib, but in windows archive no similar file. Same with libfbjni.so and libfbjni.dylib. Could you help me? Maybe i do something wrong.

Thanks!

cc @peterjc123 @maxluk @nbcsm @guyang3532 @gunandrose4u @smartcat2010 @mszhanyi

**Comment by erip** [comment]
There are no Java bindings for Windows [yet](https://github.com/pytorch/pytorch/pull/44257). I'm currently working on this feature, and am making appropriate changes in upstream dependencies - hopefully this can be resolved soon.

**Comment by mszhanyi** [comment]
@beeqwe so far, windows binaries doesn't support java. We're working on it.

**Comment by mszhanyi** [comment]
Duplicate of #32516

**Comment by erip** [comment]
@mszhanyi I think this can be closed with the merge of #44257

**Comment by mszhanyi** [comment]
@erip. Well done!
